OSDOCS#10249: Drafted docs for creating a cluster with egress lockdown (#81456)
openshift/openshift-docs: rosa_hcp/rosa-hcp-disconnected-install.adoc

:_mod-docs-content-type: ASSEMBLY
[id="hcp-aws-disconnected-install"]
= Creating a disconnected {product-title} cluster
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: rosa-hcp-disconnected-install

toc::[]

> You can create a fully operational cluster that does not require a public Internet egress by configuring a virtual private cloud (VPC) and using the `--properties zero_egress:true` flag when creating your cluster.

Reviewer: ```suggestion
You can create a fully operational cluster that does not require a public internet egress by configuring a virtual private cloud (VPC) and using the `--properties zero_egress:true` flag when creating your cluster.
```

.Prequisites

* You have an AWS account with sufficient permissions to create VPCs, subnets, and other required infrastructure.

> * You have installed Terraform v1.4.0+ CLI.

Reviewer: ```suggestion
* You have installed the Terraform v1.4.0+ CLI.
```

> * You have installed ROSA v1.2.45+ CLI.

Reviewer: ```suggestion
* You have installed the ROSA v1.2.45+ CLI.
```

* You have installed and configured the AWS CLI with the necessary credentials.
* You have installed the git CLI.

[id="rosa-hcp-creating-vpc-disconnected-install"]
== Creating a Virtual Private Cloud for your disconected {hcp-title} clusters

> You must have a Virtual Private Cloud (VPC) to create {hcp-title} cluster. You can use the following methods to create a VPC:

Reviewer: ```suggestion
You must have a Virtual Private Cloud (VPC) to create {hcp-title} clusters. You can use one of the following methods to create a VPC:
```
